Changes

```diff
@@ -406,6 +406,11 @@ Next major version (4.05.0):
   (François Bobot, review by Gabriel Scherer, Xavier Leroy, Damien Doligez,
    Frédéric Bour)
 
+* GPR#1010: change the compilation scheme of the stdlib. Now x.ml in
+  compiles to stdlib__x.cm* and we add a Stdlib alias module that is
+  opened by default to get back the short names.
```

Comment:
This is a big change, I think it should be described in terms of what the user now sees, rather than just how it works. Perhaps:

```
* GPR#1010: pack all standard library modules into a single module Stdlib
  which is the default opened module (Stdlib itself includes Pervasives) to free
  up the global namespace for other standard libraries, while still allowing any
  OCaml standard library module to be referred to as Stdlib.Module). This is
  implemented efficiently using module aliases (prefixing all modules with
  Stdlib__, e.g. Stdlib__string).`